src/web/app/components/question-types/question-edit-answer-form/text-question-edit-answer-form.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="plain-text-area" *ngIf="!questionDetails.shouldAllowRichText">
   <textarea [ngModel]="responseDetails.answer" (ngModelChange)="triggerResponseDetailsChange('answer', $event)" [attr.aria-label]="getAriaLabel()" [disabled]="isDisabled"></textarea>
 </div>
-<tm-rich-text-editor *ngIf="questionDetails.shouldAllowRichText" [attr.aria-label]="getAriaLabel()" [richText]="responseDetails.answer" (richTextChange)="triggerResponseDetailsChange('answer', $event)" [isDisabled]="isDisabled"></tm-rich-text-editor>
+<tm-rich-text-editor *ngIf="questionDetails.shouldAllowRichText" [richText]="responseDetails.answer" (richTextChange)="triggerResponseDetailsChange('answer', $event)" [isDisabled]="isDisabled"></tm-rich-text-editor>
```

Comment:
Can I check why this `aria-label` was removed?

Reply:
Sure, this is the error thrown `
[tm-rich-text-editor[aria-label="Response for Teammates Test2"]]
Fix all of the following:
  aria-label attribute cannot be used on a tm-rich-text-editor with no valid role attribute` in testrun `[#12279] Instructor home page: Improve display of card header on mobile #1016`. Deleting this aria-label resolved the error, but I must admit that I'm not 100% sure why this error was thrown in the first place.
What do you think about the error?
[test_log.txt]
The whole log: (https://github.com/TEAMMATES/teammates/files/12640173/test_log.txt)